test/flexible_type/flexible_datatype.cxx

```diff
@@ -1,25 +1,3 @@
-/*
```

Comment:
We should probably replace this with our new header.

Reply:
done